src/components/flowchart/flowchart.js

```diff
@@ -220,22 +220,28 @@ export class FlowChart extends Component {
 
     if (changed('edges', 'nodes', 'layers', 'chartSize', 'clickedNode')) {
```

Comment:
Hey, it looks like this section of code is for different beahviour of the stand-alone app. Perhaps having a comment here could help clarify the purpose, so we'll have better context if we refactor this component in the future. What do you think?

Reply:
@Huongg Not exactly, those condition is also being used in normal mode, but yes refactoring `flowchart.js` component will really help overall.